src/components/ListedSpecies/BandingSection.jsx

```diff
@@ -3,12 +3,14 @@ import {
   Button,
   Checkbox,
   Code,
+  Flex,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Flex' is defined but never used.